# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tmux/default-history-limit.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- tmux-3.5a/options-table.c.vanilla	2025-06-17 14:24:40.064369957 +0200
+++ tmux-3.5a/options-table.c	2025-06-17 14:25:07.013000000 +0200
@@ -564,7 +564,7 @@
 	  .scope = OPTIONS_TABLE_SESSION,
 	  .minimum = 0,
 	  .maximum = INT_MAX,
-	  .default_num = 2000,
+	  .default_num = 2500,
 	  .unit = "lines",
 	  .text = "Maximum number of lines to keep in the history for each "
 		  "pane. "
